src/NRedisStack.Core/Bloom/BloomCommands.cs

```diff
@@ -68,15 +75,64 @@ public RedisResult Info(RedisKey key)
             return _db.Execute(BF.INSERT, args);
         }
 
-        public RedisResult ScanDump(RedisKey key, int iterator)
+        public RedisResult LoadChunk(RedisKey key, int iterator, RedisValue data)
         {
-            return _db.Execute(BF.SCANDUMP, key, iterator);
+            return _db.Execute(BF.LOADCHUNK, key, iterator, data);
         }
 
-        public RedisResult LoadChunk(RedisKey key, int iterator, RedisValue data)
+        public bool[] MAdd(RedisKey key, RedisValue[] items)
         {
-            return _db.Execute(BF.LOADCHUNK, key, iterator, data);
+            if (items == null)
+                throw new ArgumentNullException(nameof(items));
+
+            List<object> args = new List<object> { key };
+
+            foreach (var item in items)
```

Comment:
Any reason you can't concat these two types of lists, rather than loop? Is it possible to do a:

args.Concat(items).ToList

Reply:
"Concat" and "AddRange" only works for Enumerable, and RedisValue[] is not Enumerable (it's also not working for input of type `List<RedisValue>`).

I haven't found a more elegant way than this..
I can Change it to:`items.ForEach(item => args.Add(item));`
Tell me if it's better in your opinion